### ISSUE TITLE
convert_to('PARMED') super slow

### ISSUE THREAD

**Issue by fhh2626** [MDAnalysis/mdanalysis#3028]
## Expected behavior ##

The call of the method should be finished within a reasonable time scale

## Actual behavior ##

For a 10k-atom system, this function call takes 2-3 min, and for a 60k-atom system, this takes > 15 min.

## Code to reproduce the behavior ##

``` python
import MDAnalysis as mda
import parmed as pmd
prm = pmd.load_file('whatever.parm7', 'whatever.rst7')
u = mda.Universe(prm)
prot = u.select_atoms('all')
prm_u = prot.convert_to('PARMED')

....

```

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
the latest conda version of MDAnalysis and parmed
- Which version of Python (`python -V`)?
Python 3.7.8
- Which operating system?
I tried both Windows 10 and Linux Mint 19

**Comment by IAlibay** [comment]
Just re-opening to confirm that sufficient work has been done here to address the issue.

@fhh2626 would you be able to confirm that the latest fix (#3029) alleviates your issue? (this would involve installing the developer version of MDAnalysis).

**Comment by fhh2626** [comment]
Yes, thanks for your nice work!
